packages/scaffold/src/views/w3m-connect-view/index.ts

```diff
@@ -200,15 +200,17 @@ export class W3mConnectView extends LitElement {
       return null
     }
 
-    const roundedCount = Math.floor(ApiController.state.count / 10) * 10
+    const rawCount = ApiController.state.count
+    const roundedCount = rawCount < 10 ? rawCount : Math.floor(rawCount / 10) * 10
```

Comment:
@saintmaxi @glitch-txs This logic will only put a `+` prefix for the numbers between 10 - 20. But if it's higher than 20, it will just render the number. Should this be discussed design-wise again? 

Not sure what is expected logic here is.

Reply:
Mmm let's say

const rawCount = 31

then

```ts
const roundedCount = Math.floor(31/10) * 10
// 30
```
This means  roundedCount < rawCount = true

Therefore it would get the + sign as 30+

Am I missing anything? 🤔